src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadManifestUpdater.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.Extensions.EnvironmentAbstractions;
 using Microsoft.NET.Sdk.WorkloadManifestReader;
+using WorkloadCollection = System.Collections.Generic.Dictionary<Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadId, Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadDefinition>;
```

Comment:
```suggestion
using WorkloadCollection = Dictionary<WorkloadId, WorkloadDefinition>;
```

Reply:
That's invalid. Using statements don't have the context of other using statements or namespaces unless they are declared inside a namespace.